File: practice17/10.cpp
//https://www.nowcoder.com/practice/22d33f7d65b84a759babae4001e87740?tpId=90&tqId=30885&tPage=6&rp=6&ru=/ta/2018test&qru=/ta/2018test/question-ranking
#include<iostream>
#include<string>
using namespace std;
int main()
{
	string str;
	char ch;
	int i, j, count;
	while (cin >> str)
	{
		for (i = 0; i < str.size(); i++)
		{
			ch = str[i];
			count = 0;
			for (j = i; j < str.size(); j++)
			{
				if (str[j] == ch)
				{
					count++;
				}
				else
				{
					break;
				}
			}
			if (count == 3)
			{
				str.erase(str.begin() + i, str.begin() + j);
				i = -1;
			}
		}
		cout << str << endl;
	}
	system("pause");
	return 0;
}

File: practice28/10.java


//https://www.nowcoder.com/practice/02f2bbaadbcf424b8cbc7e264ddef9b4?tpId=98&tqId=33056&tPage=12&rp=12&ru=/ta/2019test&qru=/ta/2019test/question-ranking
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int[] pt = new int[n];
        for (int i = 0; i < n; i++) {
            pt[i] = scanner.nextInt();
        }
        int count = n - 1;
        for (int i = n - 1; i >= 0; i--) {
            if (i > 0 && pt[i] > pt[i - 1]) {
                count--;
            } else {
                break;
            }
        }
        System.out.println(count);
    }
}
